paper.md

```diff
@@ -35,65 +35,75 @@ bibliography: paper.bib
+(Git, https://git-scm.com), and adopting ideas and procedures from software distributions, DataLad delivers a completely open, pioneering platform for flexible decentralized research data management (dRDM) [@Hanke_2021].
 
 # Statement of Need
 
 Code, data, and computing environments are at the core of scientific practice, and unobstructed access and efficient management of all those digital objects promotes scientific discovery through collaboration, reproducibility, and replicability. 
-While code management and sharing is streamlined with the advance of software distributions, distributed version control systems, and social coding portals like GitHub, data has remained a “2nd-class citizen” in the contemporary scientific process, despite FAIR principles postulating demands on public data hosting portals and the big callout for Data Science.
-Disconnected FAIR data hosting portals provide a cacophony of data access and authentication methods, data versioning is frequently ignored, and shared data provenance is often not recoverable simply because data management is rarely considered to be an integral part of the scientific process.
-DataLad aims to solve these issues by streamlining data consumption, publication, and updating routines, by providing a simplified core API for git and git-annex operations, and by providing additional features for reproducible science such as command execution and automatic recomputation based on complete process provenance.
-As it is interoperable with a large variety of scientific and commercial hosting services and available for all operating systems, DataLad can be integrated into established systems with minimal adjustments.
-
-## Why git and git-annex
-
-Git is an excellent distributed content management system geared towards managing and collaborating on text files, but as such it cannot handle large or binary files well.
-Moreover, any data committed to git becomes available to all clones of that repository. In the context of scientific data, this makes it hard or impossible to provide distributed storage of managed data, or revoke individual files, such as accidentally stored personal data or data from participants that withdrew from a study.
-Git-annex takes advantage of git's ability to manage textual information to overcome git's limitation in managing individual file content that is either large or sensitive.
-File content managed by git-annex is placed into an annex, and in its place, git-annex commits a symbolic link pointing to the content of the file into Git.
+While code management<!-- maybe "software development" to use a more intuitive term--> and sharing are streamlined with the advance of software distributions, distributed version control systems, and social coding portals like GitHub, data have remained a “2nd-class citizen” in the contemporary scientific process, despite FAIR principles postulating demands on public data hosting portals and the big callout for Data Science.
```

Comment:
agree, especially since we go into 'sharing' and distributions, thus

```suggestion
While software development and sharing are streamlined with the advance of software distributions, distributed version control systems, and social coding portals like GitHub, data have remained a “2nd-class citizen” in the contemporary scientific process, despite FAIR principles postulating demands on public data hosting portals and the big callout for Data Science.
```